src/release.js

```diff
@@ -447,7 +475,7 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
                 log.done(`'${releaseBranch}' merged into '${branchPrefix}-${data.version}'.`);
             } catch (err) {
```

Comment:
question: what happens if this tool exists half-way? E.g. if because of merge conflicts or connection fail it won't be able to merge `master` back into `develop`.
I don't see exit code or anything, just `log.exit(An error occurred: ${err});` below, not even `log.error`. 
In this case, as I understand, user will need to complete release manually. 
Will he be somehow notified that release failed?